### PR TITLE
749: Translations: Open the first editor if the current table has only one

### LINKED ISSUE #749: Open the string if is a page with only one
Example case, I am in the permalink page of the strings but I need to click to open and show it.

### PR DESCRIPTION
**Pull request by ocean90**
Fixes #749.

### CODE REVIEW

File: assets/js/editor.js [hunk 1]
```diff
@@ -6,9 +6,17 @@ $gp.editor = (
...
+				var $previewRows;
+
 				$gp.init();
 				$gp.editor.table = table;
 				$gp.editor.install_hooks();
+
+				// Open the first editor if the current table has only one.
+				$previewRows = $gp.editor.table.find( 'tr.preview' );
+				if ( 1 === $previewRows.length  ) {
+					$previewRows.eq( 0 ).find( 'a.edit' ).trigger( 'click' );
```
Comment (toolstack) [inline assets/js/editor.js]: Should probably just call `editor.show()` instead of triggering a click.

**Review by toolstack — COMMENTED** [review]
Two general thoughts:
- Should this be done on the PHP side instead?
- Should there be a user preference to enable/disable this?

File: assets/js/editor.js (hunk 1 above)
Comment (Mte90) [reply assets/js/editor.js]: probably yes

**Comment by ocean90** [comment]
> Should this be done on the PHP side instead?

Doing it in JS makes this far easier and inits all the additional features like reverting to the default translations (#766)

> Should there be a user preference to enable/disable this?

Why would someone disable this?

**Comment by toolstack** [comment]
>Should this be done on the PHP side instead?
>>Doing it in JS makes this far easier and inits all the additional features like reverting to the default translations (#766)

I'm fine with that then.

>Should there be a user preference to enable/disable this?
>>Why would someone disable this?

Why do users want anything? ;) 

I'm not bothered either way, just wanted to ask the question.

**Review by toolstack — APPROVED** [review]
:+1:

**Pull request merged**